Civi/RemoteTools/JsonSchema/JsonSchema.php

```diff
@@ -139,6 +139,19 @@ public function __construct(array $keywords) {
     $this->keywords = $keywords;
   }
 
+  public function __clone() {
+    $this->keywords = \array_map(function ($value) {
+      if (is_object($value)) {
+        return clone $value;
+      }
+      elseif (\is_array($value)) {
+        return \array_values(\array_map(fn ($value) => is_object($value) ? clone $value : $value, $value));
```

Comment:
I think we don't need the root namespace `\` identifiers for built-in functions like `array_map()` etc.
… sorry, forgot about that in my original review

Reply:
Changed.